tdrs-backend/tdpservice/stts/serializers.py

```diff
@@ -23,18 +23,14 @@ def get_code(self, obj):
         return obj.code
 
 
-class STTUpdateSerializer(serializers.ModelSerializer):
-    """STT serializer."""
+class STTPrimaryKeyRelatedField(serializers.PrimaryKeyRelatedField):
```

Comment:
This field can be used within a serializer to allow setting an STT via the ID and retrieving it as the full object representation.